public/src/index.js

```diff
@@ -141,3 +141,34 @@ const checkSession = async () => {
 
 window.addEventListener('DOMContentLoaded', checkSession, {once: true});
 config.currentPage = config.header.main.render(config);
+
+// Регистрация Service Worker
+const registerServiceWorker = async () => {
+    if ('serviceWorker' in navigator) {
+        navigator.serviceWorker.register('/sw.js', {scope: '/'})
+            .then((registration) => {
+                const data = {
+                    type: 'CACHE_URLS',
+                    payload: [
+                        location.href,
+                        ...performance.getEntriesByType('resource').map((r) => r.name),
+                    ],
+                };
+                if (registration.installing) {
+                    registration.installing.postMessage(data);
+                    console.log('Service worker installing');
```

Comment:
```suggestion
                    console.info('Service worker installing');
```

А то спам в логи какой-то получается)

Хотя сообщение "installing" я бы вообще выпилил, не особо вижу в нем смысла. Интересен результат же